app/js/views/services-list-item.jsx

```diff
@@ -38,19 +38,12 @@ export default class ServicesListItem extends React.Component {
     return `hsla(${h},${s}%,50%,${l})`;
   }
 
-  render() {
-    let serviceType = 'Unknown service';
-    let icon = 'unknown';
-
-    switch (this.props.type) {
-      case 'Extended color light':
-        serviceType = 'Light';
-        icon = 'light';
-        break;
-    }
+  renderLightService(isConnected) {
```

Comment:
Don't have better idea right now, how to split or generalize service item rendering without big changes (I'd defer something big for later). What do you think?